### PR TITLE
Feature/CVSB-112

### PR DESCRIPTION
**Pull request by danieliordache**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "generate-env": "node ./config/scripts/replace.env",
     "preionic:serve": "npm run-script generate-env --dev",
     "ionic:serve": "ionic-app-scripts serve --nobrowser",
-    "prebuild-ios": "npm run-script generate-env --dev",
-    "build-ios": "ionic-app-scripts build --target cordova --platform ios --prod --aot --minifyjs -- minifycss --optimizejs",
+    "prebuild-ios": "npm run-script generate-env --pipeline",
```
Comment (adpetrescu) [inline package.json]: why is this change required?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
       // statusBar.styleDefault();
       statusBar.overlaysWebView(true);
       statusBar.styleLightContent();
       splashScreen.hide();
 
       // Mobile accessibility
-      if (platform.is('cordova')) {
-        this.accessibilityFeatures();
-      }
+      // if (platform.is('cordova')) {
```
Comment (adpetrescu) [inline src/app/app.component.ts]: can you please remove comments?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
+      //   this.accessibilityFeatures();
+      // }
 
       // Resuming app from background Mobile Accessibility
-      platform.resume.subscribe(() => {
-        this.accessibilityFeatures();
-      });
+      // platform.resume.subscribe(() => {
```
Comment (adpetrescu) [inline src/app/app.component.ts]: can you please remove comments?

File: package.json (hunk 1 above)
Comment (danieliordache) [reply package.json]: done

File: src/app/app.component.ts (hunk 2 above)
Comment (danieliordache) [reply src/app/app.component.ts]: done

File: src/app/app.component.ts (hunk 3 above)
Comment (danieliordache) [reply src/app/app.component.ts]: done

File: src/pages/testing/defects/add-defect/add-defect.ts [hunk 4]
```diff
@@ -87,9 +93,13 @@ export class AddDefectPage {
       advisory: advisory,
       isEdit: false
     });
+    this.events.publish('navToDetails');
```
[the remark this answers is not in the document]
Comment (danieliordache) [reply src/pages/testing/defects/add-defect/add-defect.ts]: done

File: src/pipes/search/search.pipe.ts [hunk 5]
```diff
@@ -1,39 +1,41 @@
...
+            } else {
+              continue;
+            }
           }
         }
       }
     );
-    return this.sortAlphabetically(filteredArray, properties[0]);
+    return filteredArray
+    // return this.sortAlphabetically(filteredArray, properties[0]);
```
[the remark this answers is not in the document]
Comment (danieliordache) [reply src/pipes/search/search.pipe.ts]: done

File: src/providers/defects/defects.service.spec.ts [hunk 6]
```diff
@@ -9,7 +11,20 @@ describe('Provider: DefectsService', () => {
...
+    categories: DefectsDataMock.DefectsData
+  };
+  const defectsItems: DefectItemModel[] = DefectsDataMock.DefectsData[0].items
+  const defectsDeficiencies: DefectDeficiencyModel[] = DefectsDataMock.DefectsData[0].items[0].deficiencies
+
+
   beforeEach(() => {
+    spy = jasmine.createSpyObj('StorageService', {
+      'read': new Promise(resolve => {
+        return []
```
[the remark this answers is not in the document]
Comment (danieliordache) [reply src/providers/defects/defects.service.spec.ts]: done
